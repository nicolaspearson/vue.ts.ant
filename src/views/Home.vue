<template>
  <div class="home">
    <Logo />
    <HelloWorld msg="Welcome to Your Vue.js + TypeScript App" />
  </div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

import Logo from '@/assets/logo.svg';
import HelloWorld from '@/components/HelloWorld.vue';

@Component({
  components: {
    Logo,
    HelloWorld,
  },
})
export default class Home extends Vue {}
</script>
