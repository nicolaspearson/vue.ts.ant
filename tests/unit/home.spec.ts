import { shallowMount } from '@vue/test-utils';

import Home from '@/views/Home.vue';

describe('Home.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Home, {
      propsData: {},
      stubs: ['Logo'],
    });
    expect(wrapper).toMatchSnapshot();
  });
});
