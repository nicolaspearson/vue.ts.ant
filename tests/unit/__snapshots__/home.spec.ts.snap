// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue matches snapshot 1`] = `
<div class="home">
  <h1>Welcome to Your Vue.js + TypeScript App</h1>
  <section class="logo">
    <logo-stub></logo-stub>
  </section>
</div>
`;
